src/routes/fest-route/fest-discussion.jsx

```diff
@@ -0,0 +1,102 @@
+import { useMutation } from "@apollo/client";
+import { useContext, useState } from "react";
+import { DiscussionCard } from "../../components";
+import { Button } from "../../components/button/index";
+import { CREATE_DISCUSSION, GET_DISCUSSIONS } from "../../graphql/index.js";
+import { CurrentUserContext } from "../../store/current-user-context.js";
+
+export const FestDiscussion = ({ discussionQuery, festQuery, festId }) => {
```

Comment:
Page needs to be responsive on smaller screen sizes (large and below)

Reply:
I cannot seem to find in Figma what it should look like.

Reply:
I adjusted it to be 1/2 of the screen size for "small" screen size (501px - 1169px). There will need to be some edits made to the overall fest-route at some point to make sure that entire route is responsive. That might be outside the scope of this ticket, and I don't want to mess with it since it might impact other people's work.

Reply:
For large screen sizes and above it should look somewhat like the figma. As far as medium and below I'm not entirely sure how it should look. I know that everything should fit on the page and cause no horizontal scrolling so it's up to you if you want to tackle that. We can always make an improvement ticket for it when we have more information about how it should look

Reply:
![image](https://github.com/jahorwitz/slopopedia/assets/110426130/c5d995f8-1972-43c5-8e0d-8dee5b08cb56)
Large(1024px) should have either margin or padding to keep it from touching the sides
![image](https://github.com/jahorwitz/slopopedia/assets/110426130/94d3143f-c0da-4996-8fd5-213195d895c5)
Should look similar to this with smaller margins

Reply:
Oh you are referring to the fest-route itself. Yeah that's been that way before this ticket. I don't think the spacing on the side was added when created. Since it's beyond the scope of this ticket (which is only to create the fest-discussion route), I think we should make a ticket. I'm happy to pick that up as my next one. Any thoughts on that?

Reply:
Oh okay I didn't realize that it was the entire route. That's fine then, I'll make a ticket for that later.

Reply:
Cool! When you create it can you assign it to me? I would love to get some tailwind practice on responsiveness. I think I will try to refractor the entire route to make it more scalable too.

Reply:
Yeah I can do that